app/index.html

```diff
@@ -131,6 +131,7 @@
     <script src="scripts/directives/gaugechart.js"></script>
     <script src="scripts/directives/passratechange.js"></script>
     <script src="scripts/directives/passratetime.js"></script>
+    <script src="scripts/directives/pupilteacherratiochart.js"></script>
```

Comment:
can we move these `scripts/directives/*` up with the other directives at line 104?